examples/agents/agents_schelling3d.jl

```diff
@@ -43,5 +43,4 @@ using InteractiveDynamics, GLMakie
 
 model = initialize()
 ac(agent) = (:red, :blue)[agent.group]
-am(agent) = (:circle, :rect)[agent.group]
-fig, step = abm_play(model, agent_step!, dummystep; ac, am, as = 5000)
+fig, abmstepper = abm_play(model, agent_step!, dummystep; ac, as = 0.5)
```

Comment:
Why were the markers removed from this example? It worked for me (I could see both marker types)

Reply:
I've removed it because it threw errors for me. Only the spheres worked but cubes seemed to cause the errors. Really not sure why that's the case. So I removed it (at least for now) so I could test things out.

And `as=5000` was way off so I reduced it to 0.5 to actually be able to see anything. :D

Reply:
yeah there was a mixup in the code: in my version I have _removed_ the line that converts `:circle` to `Sphere`. In this case, I needed super large sizes because 3D plotting in makie is still very wonky.

Reply:
Ah OK, should I also remove the line then?

Reply:
The problem for me was that `Sphere` by itself was not working... But for you it is? Let's leave it as is for now.